site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+
+=== User Report
+
+You can also view a list of all users with breached passwords. To do so, click on the *Breached users* search button on the [breadcrumb]#Reactor# tab of the administrative user interface. You'll see this report:
+
+image::guides/breached-password-detection/user-report.png[Individual password breach report,width=700,role=box-shadow margin-right]
+
+This is a paginated list of users with compromised credentials. You can then manage the user or lock their account. If you navigate to the user's details page, you see a warning about their vulnerable password, based on the match type of the compromised password:
+
+image::guides/breached-password-detection/admin-user-vulnerable-message.png[The vulnerable password message in the admin interface,width=700,role=box-shadow margin-right]
+
+You'll also be able to modify the account. Options include locking their account, forcing a password change or resetting their password and sending them an email:
+
+image::guides/breached-password-detection/acting-on-a-user.png[Acting on a user,width=700,role=box-shadow margin-right]
+
+=== Custom Reporting
+
+If you have custom reporting needs, such as querying on the number of users with breached passwords over time, or finding the number of users with multiple password breaches, ingest the data into your own analytics system.
+
+The best method to do so is to set up a webhook to listen for the link:../events-webhooks/events#user-password-breach[`user.password.breach` event]. 
+
+You can read more about setting up link:../events-webhooks/[webhooks here].
+
+== End User Experience
+
+What does the end user experience when a breached password is detected? It depends on whether they are registering or logging in when the breach is detected.
+
+It also depends on whether you are using the FusionAuth provided page templates or have built your own integration using the link:../apis/[APIs].
+
+=== FusionAuth Provided Templates
+
+If the user is registering or logging in with the provided page templates, they'll see this on registration, should they choose a password the system determines has been compromised:
+
+image::guides/breached-password-detection/oauth-registration-breached-message.png[What a user who is registering with a breached password will see.,width=700,role=box-shadow margin-right]
+
+They will not be able to complete a registration without choosing a more secure password.
+
+If you enable breach detection on login, but do not require a password change, the end user will see no difference between signing in with a secure or compromised password. If you enable sending an email, the user will receive that notice.
+
+If you, on the other hand, require a password change, a user signs in with a password that is found to have been compromised, they'll see:
+
+image::guides/breached-password-detection/oauth-login-breached-message.png[What a user who is signing in with a breached password will see.,width=700,role=box-shadow margin-right]
```

Comment:
Would it be more consistent to not chop off top of the browser on this one?

Reply:
Make sense.